asterius/rts/rts.mblockalloc.mjs

```diff
@@ -1,13 +1,25 @@
 import * as rtsConstants from "./rts.constants.mjs";
 import { Memory } from "./rts.memory.mjs";
+import * as assert from "assert";
```

Comment:
This is a node-only import which will break the browser target again. We can have a local no-op assert function here for now, before we figure out how to properly do rts assertions.